standard/standard_4.php

```diff
@@ -181,13 +181,16 @@ function unserialize(string $data, array $options = []): mixed {}
 /**
  * Dumps information about a variable
  * @link https://php.net/manual/en/function.var-dump.php
- * @param mixed $value <p>
- * The variable you want to export.
- * </p>
+ * @param mixed $value The variable you want to export.
  * @param mixed ...$values [optional]
+ * @param mixed ...$vars [optional]
```

Comment:
Why `...$vars`? Can't see it anywhere:
- https://www.php.net/manual/en/function.var-dump.php 
- https://github.com/php/php-src/blob/master/ext/standard/basic_functions.stub.php#L1907
- https://github.com/php/php-src/blob/master/ext/standard/var.c#L217